core/station/impl/src/repositories/indexes/request_policy_resource_index.rs

```diff
@@ -371,4 +403,58 @@ mod tests {
             assert_eq!(policies[0], expected_method_ids.pop().unwrap());
         }
     }
+
+    #[test]
+    fn test_find_external_canister_call_policies_by_execution_and_validation_method() {
```

Comment:
Could we merge this test with the above one to avoid duplicating the common setup?